doc/source/whatsnew/v0.20.0.txt

```diff
@@ -50,6 +50,7 @@ Other enhancements
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
 
+- pd.cut and qcut now support datetime64 and timedelta64 dtypes (issue:`14714`)
```

Comment:
@aileronajay yes, this is rst (restructuredText) used by sphinx to generate the html docs: http://www.sphinx-doc.org/en/1.4.9/rest.html